FhirToDataLake/deploy/templates/ContainerApp/DeployFhirPipelineToContainerApp.json

```diff
@@ -1,420 +1,447 @@
+          }
+      },
+      "requiredTypes":{
+          "type": "string",
+          "defaultValue": "",
+          "metadata": {
+              "description": "For data filtering use. Specify which types of resources will be included. For example, type=Patient would return only patient resources. All resource types will be exported if not specified. Leave it blank if no filter is applied."
+          }
+      },
+      "typeFilters":{
+          "type": "string",
+          "defaultValue": "",
+          "metadata": {
+              "description": "For data filtering use. Use along with the requiredTypes configuration. The value is a comma-separated list of FHIR queries that further restrict the results. All data of requiredTypes will be exported if not specified. Leave it blank if no filter is applied."
+          }
+      },
+      "image": {
+          "type": "string",
+          "defaultValue": "healthplatformregistry.azurecr.io/healthdatatodatalaketool:v0.6.0",
+          "metadata": {
+              "description": "Container image to deploy. Should be of the form repoName/imagename:tag for images stored in public Docker Hub, or a fully qualified URI for other registries. Images from private registries require additional registry credentials."
+          }
+      },
+      "maxInstanceCount": {
+          "type": "int",
+          "defaultValue": 10,
+          "minValue": 1,
+          "maxValue": 30,
+          "metadata": {
+              "description": "Maximum number of replicas running for fhir to datalake container app."
+          }
+      },
+      "storageAccountType": {
+          "type": "string",
+          "defaultValue": "Standard_LRS",
+          "allowedValues": [
+              "Standard_LRS",
+              "Standard_GRS",
+              "Standard_RAGRS"
+          ],
+          "metadata": {
+              "description": "Storage Account type"
           }
-        }
       }
     },
-    {
-        "type": "Microsoft.Storage/storageAccounts/providers/roleAssignments",
-        "apiVersion": "2018-09-01-preview",
-        "name": "[concat(variables('storageAccountName'), '/Microsoft.Authorization/', guid(uniqueString(variables('storageAccountName'), variables('pipelineName'), 'blob')))]",
-        "dependsOn": [
-            "[resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName'))]",
-            "[resourceId('Microsoft.App/containerApps', variables('pipelineName'))]"
-        ],
+    "variables": {
+      "pipelineContainerName": "fhir-to-datalake",
+      "pipelineName": "[toLower(parameters('pipelineName'))]",
+      "analyticsWorkspaceName": "[concat(variables('pipelineName'), '-LogAnalytics')]",
+      "environmentName": "[if(empty(parameters('containerAppEnvironmentName')), concat(variables('pipelineName'), '-Environment') ,parameters('containerAppEnvironmentName'))]",
```

Comment:
Is this a fix for assigning the RBAC for existing Container App Environment?

Reply:
It is not a fix. actually, I don't modify it. it maybe a show issue of github.